k8s/pkg/registryserver/resource_cache/ns_cache.go

```diff
@@ -3,16 +3,17 @@ package resource_cache
 import (
 	"github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis/networkservice/v1"
 	"github.com/networkservicemesh/networkservicemesh/k8s/pkg/networkservice/informers/externalversions"
+	"sync"
 )
 
 type NetworkServiceCache struct {
 	cache           abstractResourceCache
-	networkServices map[string]*v1.NetworkService
+	networkServices sync.Map
 }
 
 func NewNetworkServiceCache() *NetworkServiceCache {
 	rv := &NetworkServiceCache{
-		networkServices: make(map[string]*v1.NetworkService),
+		networkServices: sync.Map{},
```

Comment:
No need to override value from structure.

Reply:
oops! Thanks, fixed